dependencies.props

```diff
@@ -28,6 +28,8 @@
     <SystemServiceModelNetTcpPackageVersion>$(SystemServiceModelDuplexPackageVersion)</SystemServiceModelNetTcpPackageVersion>
     <SystemServiceModelPrimitivesPackageVersion>$(SystemServiceModelDuplexPackageVersion)</SystemServiceModelPrimitivesPackageVersion>
     <SystemServiceModelSecurityPackageVersion>$(SystemServiceModelDuplexPackageVersion)</SystemServiceModelSecurityPackageVersion>
+
+    <NuGetProjectModelPackageVersion>4.3.0-preview2-4095</NuGetProjectModelPackageVersion>
```

Comment:
This should go in the `infrastructure and test only dependencies` property group down below, just for categorization.